src/domain/events/index.ts

```diff
@@ -206,6 +212,23 @@ export function add(
     // Iterate over the conditionalOrders to make sure that the params are not already in the registry
     for (const conditionalOrder of conditionalOrders?.values() ?? []) {
       // Check if the params are in the conditionalOrder
+      if (conditionalOrder) {
```

Comment:
maybe add a TODO to delete these lines, and schedule the deletion. I understand you only wrote this to see how often it was happening we were adding an order when it was not time

Reply:
Added